index.html

```diff
@@ -42,18 +42,17 @@
     <section id="abstract" class="informative">
       <p>
         This specification defines an API allowing <a data-cite=
-        "appmanifest">web applications</a> to set an application-wide badge,
+        "appmanifest#installable-web-applications">web applications</a> to set an application-wide badge,
         shown in an operating-system-specific place associated with the
         application (such as the shelf or home screen), for the purpose of
         notifying the user when the state of the application has changed (e.g.,
         when new messages have arrived), without showing a more heavyweight
-        <a data-cite="NOTIFICATIONS">notification</a>.
+        <a data-cite="notifications#concept-notification">notification</a>.
       </p>
     </section>
     <section id="sotd">
       <p>
-        This specification is currently a work in progress. Implementors should
-        not expose the API surface to the open web.
+        This is an early draft of the Badging API spec.
```

Comment:
Not really sure what to put here? It’s not in the w3c so it’s not a spec yet, but it’s not really a draft either.

(I’ve copied this from the WebShareTarget spec, mostly I just wanted it to show up as changed in the diff).